ksqldb-rest-app/src/main/java/io/confluent/ksql/api/impl/QueryEndpoint.java

```diff
@@ -77,6 +80,7 @@ public QueryEndpoint(
   ) {
     this.ksqlEngine = ksqlEngine;
     this.ksqlConfig = ksqlConfig;
+    this.ksqlRestConfig = new KsqlRestConfig(ksqlConfig.originals());
```

Comment:
Nit: You can just pass this in from `KsqlServerEndpoints` -> `KsqlRestApplication` if you wanted.  That's a little more consistent with how we usually create it.